README.md

```diff
@@ -303,7 +363,16 @@ $dataset
 WW_TuneCP5_13p6TeV-pythia8
 ```
 
-:exclamation: Do not make the file list greater than 4k files to avoid scaleout issues in various site
+
+## Get processed information
+
+Get the processed run & luminosity for the processed events from the coffea. When you use `--skipbadfiles`, the submission will ignore files not accesible(or timeout) by `xrootd`. This script helps you to dump the processed luminosity into json which can be calculated by `brilcalc` tool and provide lists of files failed to processed by comparing to original json inputs from the `.coffea` files.
```

Comment:
et the processed run & luminosity for the processed events from the coffea. -> Get the run & luminosity information for the processed events from the coffea output files.

accesible(or timeout) -> accessible (or time out)

into json -> into a json file

provide lists of files failed to processed by comparing to original json inputs from the `.coffea` files. -> provide a list of failed lumi sections by comparing the original json input to the one from the `.coffea` files.